internal/pkg/describe/status.go

```diff
@@ -341,6 +335,395 @@ func (a *apprunnerServiceStatus) HumanString() string {
+	fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+	// Write summary of task definition revision.
+	desiredTaskDefVersion, err := awsECS.TaskDefinitionVersion(s.Service.TaskDefinition)
+	if err == nil && s.Service.RunningCount > 0 {
+		header := "Deployment"
+		data := s.taskDefinitionRevisionData()
+		bar := summaryBar(
+			[]int{
+				data[desiredTaskDefVersion],
+				(int)(s.Service.RunningCount) - data[desiredTaskDefVersion],
+			},
+			[]string{color.Green.Sprint("■"), color.Yellow.Sprint("□")})
+		stringSummary := fmt.Sprintf("%d/%d Running task definition version %d (desired)",
+			data[desiredTaskDefVersion],
+			s.Service.RunningCount,
+			desiredTaskDefVersion)
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+	}
+
+	// Write summary of HTTP health and container health.
+	if s.shouldShowHealthSummary() {
+		header := "Healthy"
+		if s.shouldShowHTTPHealth() {
+			healthyCount := s.healthyHTTPTasksCount()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes HTTP Health Checks", healthyCount, s.Service.RunningCount)
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+			header = ""
+		}
+
+		if s.shouldShowContainerHealth() {
+			healthyCount, _, _ := s.containerHealthData()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes Container Health Checks", healthyCount, s.Service.RunningCount)
+
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+		}
+	}
+
+	// Write summary of capacity providers.
+	if s.shouldShowCapacityProvider() {
+		header := "Capacity Provider"
+		fargate, spot, unset := s.capacityProviderData()
+		bar := summaryBar([]int{fargate, spot, unset}, []string{color.Blue.Sprint("f"), color.Magenta.Sprint("s"), color.Grey.Sprint("d")})
+		var cpSummaries []string
+		if fargate != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate", fargate, s.Service.RunningCount))
+		}
+		if spot != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate Spot", spot, s.Service.RunningCount))
+		}
+		if unset != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on default capacity provider", unset, s.Service.RunningCount))
+		}
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, strings.Join(cpSummaries, ", "))
+	}
+	writer.Flush()
+}
+
+func (s *ecsServiceStatus) writeStoppedTasks(writer *tabwriter.Writer) {
+	if len(s.StoppedTasks) <= 0 {
+		return
+	}
+
+	fmt.Fprint(writer, color.Bold.Sprint("\nStopped Tasks\n\n"))
+	writer.Flush()
+
+	headers := []string{"Reason", "Task IDs"}
```

Comment:
What do you think of adding `"Task Count"` here as a column?

Reply:
I forgot why I removed the column... probably because that information can be inferred from `Task IDs`.

However, reading from your next comment (limiting the number of IDs) this sounds good to me.